schema/defs-image.json

```diff
@@ -0,0 +1,99 @@
+{
+  "description": "Definitions particular to OpenContainer Image Specification",
+  "definitions": {
+    "mediaType": {
+      "id": "https://opencontainers.org/schema/image/mediaType",
+      "type": "string",
+      "pattern": "^[a-z]+/[0-9a-zA-Z.+]+$"
+    },
+    "digest": {
+      "description": "the cryptographic checksum digest of the object, in the pattern '<hash>:<hexadecimal digest>'",
+      "type": "string",
+      "pattern": "^[a-z0-9]+:[a-fA-F0-9]+$"
+    },
+    "blobObject": {
```

Comment:
ah fine. A name of a struct. Feel free to open a PR to include this in the
spec. I can update it here.

On Fri, Apr 15, 2016 at 1:27 PM Stephen Day <notifications@github.com>
wrote:

> In schema/defs-image.json
> <https://github.com/opencontainers/image-spec/pull/18#discussion_r59936055>
> :
>
> > @@ -0,0 +1,99 @@
> > +{
> > +  "description": "Definitions particular to OpenContainer Image Specification",
> > +  "definitions": {
> > +    "mediaType": {
> > +      "id": "https://opencontainers.org/schema/image/mediaType",
> > +      "type": "string",
> > +      "pattern": "^[a-z]+/[0-9a-zA-Z.+]+$"
> > +    },
> > +    "digest": {
> > +      "description": "the cryptographic checksum digest of the object, in the pattern '<hash>:<hexadecimal digest>'",
> > +      "type": "string",
> > +      "pattern": "^[a-z0-9]+:[a-fA-F0-9]+$"
> > +    },
> > +    "blobObject": {
>
> This is what we have called it from the start. The godoc is here
> <https://godoc.org/github.com/docker/distribution#Descriptor>. I am not
> sure how it never made into this specification.
>
> Other types with further fields, such as the manifest descriptor, are
> extensions to the descriptor type.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/image-spec/pull/18/files/64b5003c9985ba8714201f1ecc01460770ab5828#r59936055>
>